TROUBLESHOOTING.md

```diff
@@ -1,5 +1,68 @@
 # Troubleshooting
 
+## Logging
+
+Our libraries use the Java logging API via `java.util.logging` package.
+Configuring logging level reveals various facts that help your troubleshooting,
+including:
+
+- The timing of underlying client-server communication
+- Request and response message headers
+- Verbose messages in underlying dependency libraries
+
+While there are various ways to configure the logging,
+to quickly enable verbose logging for the Google Cloud Java libraries, create
+a file `logging.properties` with the following content:
+
+```
+# run java program pointing to this properties file with the java arg
+#   -Djava.util.logging.config.file=path/to/logging.properties
+handlers=java.util.logging.ConsoleHandler
+java.util.logging.SimpleFormatter.format=%1$tF %1$tT,%1$tL %4$-8s %3$-50s - %5$s %6$s%n
+
+# --- ConsoleHandler ---
+java.util.logging.ConsoleHandler.level=ALL
+java.util.logging.ConsoleHandler.formatter=java.util.logging.SimpleFormatter
+.level=INFO
+
+# Specify logging level for certain packages
+com.google.api.level=ALL
```

Comment:
Can we add notes to indicate that this is only needed for REST APIs and `io.groc.level` below is only needed for gRPC APIs?

Reply:
Added comments.

Note that even when you use gRPC library, some authentication happens in HTTP 1.1.

Reply:
> Note that even when you use gRPC library, some authentication happens in HTTP 1.1.

Thanks, that's good to know. I guess it would be mostly covered by the `com.google.auth.level=FINE` below?

Reply:
That does not show HTTP headers.